.env

```diff
@@ -39,3 +39,4 @@ API_LOGGING_ENABLED=1
 SENTRY_ENABLED=0
 SENTRY_ENVIRONMENT=ci-main
 USE_PUBLIC_LOG_DIR=0
+MARKET_ENABLED=0
```

Comment:
Looks like a typo, cause in the feature flag config it's named `FLAG_MARKET_ENABLED`

Reply:
Yes, you are right, thanks for spotting.
Wondering why that still worked 🤔